interop/client/client.go

```diff
@@ -37,6 +37,7 @@ var (
 	caFile                = flag.String("ca_file", "", "The file containning the CA root cert file")
 	useTLS                = flag.Bool("use_tls", false, "Connection uses TLS if true")
 	useALTS               = flag.Bool("use_alts", false, "Connection uses ALTS if true (this option can only be used on GCP)")
+	hsAddr                = flag.String("handshaker_service_address", "", "ALTS handshaker gRPC service address")
```

Comment:
Rename this to `altsHSAddr` and add work `alts` to the flag name?

Reply:
Done.